src/OrchardCore/OrchardCore.ContentManagement.GraphQL/Queries/Types/ContentItemWhereInput.cs

```diff
@@ -12,6 +12,7 @@ public class ContentItemWhereInput : WhereInputObjectGraphType
         public ContentItemWhereInput(string contentItemName)
         {
             Name =  $"{contentItemName}WhereInput";
+            DisplayName = $"{contentItemName}Where";
```

Comment:
Should we remove this? The default behaviour seems okay for input types. Specific input types can set FieldName if they want to override the default.